README.md

```diff
@@ -97,12 +97,43 @@ Kadena.Pact.ExecCommand.new()
   |> Kadena.Pact.ExecCommand.set_code(code)
   |> Kadena.Pact.ExecCommand.set_nonce(nonce)
   |> Kadena.Pact.ExecCommand.set_data(env_data)
-  |> Kadena.Pact.ExecCommand.set_metadata(keypair)
+  |> Kadena.Pact.ExecCommand.set_metadata(meta_data)
   |> Kadena.Pact.ExecCommand.add_keypair(keypair)
   |> Kadena.Pact.ExecCommand.add_signers(signers_list)
   |> Kadena.Pact.ExecCommand.build()
 ```
 
+### Continuation Command
```

Comment:
@Odraxs The documents are incomplete, please complete them with examples of each use case. As a user it is not clear to me how to build a command